sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/producer_client.hpp

```diff
@@ -20,6 +20,8 @@ namespace Azure { namespace Messaging { namespace EventHubs {
     class EventHubsPropertiesClient;
   } // namespace _detail
 
+  class ProducerClient;
+
   /**@brief Contains options for the ProducerClient creation
    */
   struct ProducerClientOptions final
```

Comment:
BTW, should these derive from Azure Core internal ClientOptions?

Reply:
I looked at the fields in ClientOptions, almost none of them are relevant to AMQP based protocols.